.buildkite/premerge.steps.yaml

```diff
@@ -0,0 +1,30 @@
+---
+common: &common
+  agents:
+    - "agent_count=8"
+    - "boot_disk_size_gb=400"
+    - "capable_of_building=platform"
+    - "machine_type=quad"
```

Comment:
This is way too big (and costly) a machine for this pipeline, better to use a `half` with `agent_count=1`.

Reply:
done